src/zope/security/checker.py

```diff
@@ -64,7 +64,7 @@ class DuplicationError(Exception):
 if os.environ.get('ZOPE_WATCH_CHECKERS'):
     try:
         WATCH_CHECKERS = int(os.environ.get('ZOPE_WATCH_CHECKERS'))
-    except ValueError:
+    except ValueError: # pragma: no cover
```

Comment:
FWIW flake8 (via pycodestyle) complains when you don't precede an inline comment with two or more spaces.

(Personally I hate that rule.)

Reply:
Yeah, I hate that rule too, and I have it turned off. It's [not](https://github.com/zopefoundation/zope.security/blame/master/src/zope/security/metaconfigure.py#L50) [like](https://github.com/zopefoundation/zope.security/blame/master/src/zope/security/metaconfigure.py#L52) [anyone](https://github.com/zopefoundation/zope.security/blame/master/src/zope/security/checker.py#L231) [actually](https://github.com/zopefoundation/zope.security/blame/master/src/zope/security/checker.py#L244) [does that](https://github.com/zopefoundation/ZODB/blame/master/src/ZODB/BaseStorage.py#L77) [consistently](https://github.com/zopefoundation/ZODB/blame/master/src/ZODB/BaseStorage.py#L78). 😉